nsxt/provider.go

```diff
@@ -42,6 +49,7 @@ type commonProviderConfig struct {
 	RetryStatusCodes       []int
 	Username               string
 	Password               string
+	LicenseDiff            listDiff
```

Comment:
Why persisting here the diffs instead of the current license list? Not sure I understand how this works.

Reply:
We want to apply/remove only the changes in license, not the whole list. So we can either persist the old+new license list, or just persist the changes